### DISCUSSION TITLE
Roy'un Facebook profili

### DISCUSSION THREAD

**Discussion by Burak2210** [sonsuzus/sonsuzus#317] [Soru Cevap] [answered]
Roy, Facebook'taki profil resmini değiştirmek istiyor. Artık Facebook'un yükleyebileceğimiz resim boyutu üzerinde bazı kısıtlamaları var.
Resmin minimum boyutu, L x L olabilir; burada L, karenin kenar uzunluğudur.

Şimdi Roy'un çeşitli boyutlarda N fotoğrafı var.
Bir fotoğrafın boyutu W x H olarak gösterilir
burada W - fotoğrafın genişliği ve H - Fotoğrafın yüksekliği

Herhangi bir fotoğraf yüklendiğinde aşağıdaki olaylar meydana gelebilir:

[1] Genişlik veya yükseklikten herhangi biri L'den küçükse, kullanıcıdan başka bir tane yüklemesi istenir. Bu durumda "BAŞKA YÜKLE" yazdırın.
[2] Genişlik ve yükseklik, her ikisi de yeterince büyükse ve
(a) fotoğraf zaten kare ise kabul edilir. Bu durumda "KABUL EDİLDİ" yazdırın.
(b) başka bir durumda kullanıcıdan onu kırpması istenir. Bu durumda "FOTOĞRAFI KIRP" yazdırın.

(alıntılar sadece açıklama amaçlıdır)

Girdi olarak L, N, W ve H verildiğinde, uygun metni çıktı olarak yazdırın.

Kısıtlamalar:
1 <= L,W,H <= 10000
1 <= N <= 1000

**Comment by DeniZZece** [comment, marked as answer]
```
#include<stdio.h>
int main()
{
	int l,n,w,h,i;
	printf("minimum kenar uzunlugunu giriniz:");
	scanf("%d", &l);
	printf("yukleyeceginiz fotograf sayisini giriniz:");
	scanf("%d",&n);
	for (i=0;i<n;i++)
	{
		printf("kenarlardan birinin uzunlugunu giriniz:");
		scanf("%d",&w);
		printf("diger kenarinin uzunlugunu giriniz:");
		scanf("%d", &h);
		if (w==h && w>=l)
		{
			printf("KABUL EDILDI\n");
		}
		else if (w>=l && h>=l)
		{
			printf("FOTOGRAFI KIRP\n");
		}
		else 
		{
			printf("BASKA YUKLE\n");
		}
	}
}
```
